### PR TITLE
Issue fixed

### PR DESCRIPTION
**Pull request by iamGpower**
email field fetched from request body, bug fix, and updated documentation

### CODE REVIEW

File: src/routes/routes.js [hunk 1]
```diff
@@ -92,7 +92,7 @@ router.get('/users/status/active',auth, newUser.getActiveUsers);
 router.get('/users/status/inactive',auth, newUser.getInActiveUsers);
 
 //get non admin users
-router.get('/users/level/intern',auth, newUser.getInternUsers);
+// router.get('/users/level/intern',auth, newUser.getInternUsers);
```
Comment (goodhands) [inline src/routes/routes.js:95]: Why did you add this again?

Comment (iamGpower) [reply src/routes/routes.js:95]: I didn't add it, just commented it out.
